src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
+    protected String deadlineBy;
+    public Deadline(String description, String deadlineBy){
+        super(description);
+        this.deadlineBy = deadlineBy;
+    }
+
+    @Override
```

Comment:
I like how you used @Override to make sure that you are properly overriding method from its parent